{
  "id": "CVE-2020-11935",
  "definition_id": "oval:com.ubuntu.trusty:def:2020119350000000",
  "title": "CVE-2020-11935 on Ubuntu 14.04 LTS (trusty) - medium.",
  "description": "It was discovered that aufs improperly managed inode reference counts in the vfsub_dentry_open() method. A local attacker could use this vulnerability to cause a denial of service attack. Mauricio Faria de Oliveira discovered that the aufs implementation in the Linux kernel improperly managed inode reference counts in the vfsub_dentry_open() method. A local attacker could use this vulnerability to cause a denial of service.",
  "note": "sbeattie\u003e upstream aufs4-linux.git commits: 515a586eeef31e0717d5dea21e2c11a965340b3c f10aea57d39d6cd311312e9e7746804f7059b5c8",
  "severity": "Medium",
  "affected": {
    "family": "unix",
    "platform": "Ubuntu 14.04 LTS"
  },
  "packages": [
    {
      "name": "linux",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-aws",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-azure",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-lts-xenial",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-meta",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-meta-aws",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-meta-azure",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-meta-lts-xenial",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-signed",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-signed-azure",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-signed-lts-xenial",
      "status": "ignored",
      "note": "out of standard support"
    }
  ],
  "references": [
    {
      "id": "CVE-2020-11935",
      "source": "CVE",
      "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-11935"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://ubuntu.com/security/CVE-2020-11935"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://ubuntu.com/security/notices/USN-4425-1"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://ubuntu.com/security/notices/USN-4426-1"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://ubuntu.com/security/notices/USN-4427-1"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://ubuntu.com/security/notices/USN-4439-1"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://ubuntu.com/security/notices/USN-4440-1"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://usn.ubuntu.com/lsn/0070-1/"
    },
    {
      "id": "",
      "source": "BUG",
      "url": "https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1873074"
    }
  ],
  "public_date": "2020-06-29T00:00:00Z",
  "public_date_at_usn": "2020-06-29T00:00:00Z",
  "discovered_by": "Mauricio Faria de Oliveira",
  "rights": "Copyright (C) 2020 Canonical Ltd."
}
